faq.html

```diff
@@ -69,22 +76,39 @@
             />
           </a>
         </li>
-        <li class="home-tab"><a href="http://realdevsquad.com/">Home</a></li>
+        <li class="home-tab">
+          <a tabindex="1" href="http://realdevsquad.com/">Home</a>
+        </li>
         <li>
-          <a class="active-tab" href="https://welcome.realdevsquad.com/">
+          <a
+            class="active-tab"
+            tabindex="1"
+            href="https://welcome.realdevsquad.com/"
+          >
             Welcome
           </a>
         </li>
-        <li><a href="https://www.realdevsquad.com/events.html">Events</a></li>
-        <li><a href="https://members.realdevsquad.com/">Members</a></li>
-        <li><a href="https://crypto.realdevsquad.com/">Crypto</a></li>
-        <li><a href="https://status.realdevsquad.com/">Status</a></li>
+        <li>
+          <a tabindex="1" href="https://www.realdevsquad.com/events.html"
+            >Events</a
+          >
+        </li>
+        <li>
+          <a tabindex="1" href="https://members.realdevsquad.com/">Members</a>
```

Comment:
Observed that most of the places we have used tabIndex="1". Can we replace it with "0" or can you provide the reason for using "1"

Reply:
it basically depend upon the scenerio what we want